all.jsonnet

```diff
@@ -47,11 +47,11 @@ local re = t.receive(commonConfig {
   hashringConfigMapName: 'hashring',
 });
 
-
 local ru = t.rule(commonConfig {
   replicas: 1,
   rulesConfig: [{ name: 'test', key: 'test' }],
   alertmanagersURLs: ['alertmanager:9093'],
+  reloaderImage: 'quay.io/openshift/origin-configmap-reloader:4.5.0',
```

Comment:
why not use the upstream image from https://github.com/jimmidyson/configmap-reload? another option would be to use [quay.io/prometheus-operator/prometheus-config-reloader](https://quay.io/repository/prometheus-operator/prometheus-config-reloader) which should provide the same features and is more robust IMHO.

Reply:
Good point. Thanks @simonpasquier Let me update to use upstream image.